frontends/mit-open/src/page-components/LearningResourceCardTemplate/LearningResourceCardTemplate.tsx

```diff
@@ -103,23 +95,162 @@ const LRCImage: React.FC<LRCImageProps> = ({
       alt=""
     />
   )
 }
 
-const variantClasses: Record<CardVariant, string> = {
-  column: "ol-lrc-col",
-  row: "ol-lrc-row",
-  "row-reverse": "ol-lrc-row-reverse",
+const LIGHT_TEXT_COLOR = "#8c8c8c"
+const SPACER = 0.75
+const SMALL_FONT_SIZE = "0.75em"
+
+const StyledCard = styled(Card)`
+  display: flex;
+  flex-direction: column;
+
+  // Ensure the resource image borders match card borders
+  .MuiCardMedia-root,
+  > .MuiCardContent-root {
+    border-radius: inherit;
+  }
+
+  .ol-lrc-details {
+    /*
+    Make content flexbox so that we can control which child fills remaining space.
+    */
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+
+    > * {
+      /*
+      Flexbox doesn't have collapsing margins, so we need to avoid double spacing.
+      The column-gap property would be a nicer solution, but it doesn't have the
+      best browser support yet.
+      */
+      margin-top: ${SPACER / 2}rem;
+      margin-bottom: ${SPACER / 2}rem;
+
+      &:first-of-type {
+        margin-top: 0;
+      }
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .ol-lrc-chip.MuiChip-root {
+    height: 2.5 * ${SMALL_FONT_SIZE};
+    font-size: ${SMALL_FONT_SIZE};
+
+    .MuiSvgIcon-root {
+      height: 1.25 * ${SMALL_FONT_SIZE};
+      width: 1.25 * ${SMALL_FONT_SIZE};
+    }
+  }
+`
+
+const StyledCardContent = styled(CardContent, {
+  shouldForwardProp: (prop) => prop !== "sortable",
+})<{
+  variant: CardVariant
+  sortable: boolean
+}>`
+  display: flex;
+  flex-direction: ${({ variant }) => variant};
+  ${({ variant }) => (variant === "column" ? "flex: 1;" : "")}
+  ${({ sortable }) => (sortable ? "padding-left: 4px;" : "")}
+`
+
+const OfferedByText = styled.span`
+  color: ${LIGHT_TEXT_COLOR};
+  padding-right: 0.25em;
+`
+
+const CardBody: React.FC<
+  Pick<LearningResourceCardTemplateProps, "resource">
+> = ({ resource }) => {
+  const offerer = resource.offered_by?.name ?? null
+  return offerer ? (
+    <div>
+      <OfferedByText>Offered by &ndash;</OfferedByText>
+      {offerer}
+    </div>
+  ) : null
 }
 
+/*
+  Last child of ol-lrc-content will take up any extra space (flex: 1) but
+  with its contents at the bottom of its box.
+  The default is stretch, we we do not want.
+*/
+const FillSpaceContentEnd = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+`
+
+const FooterRow = styled.div`
+  min-height: 2.5 * ${SMALL_FONT_SIZE}; // ensure consistent spacing even if no date
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`
+
+const TypeRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 1.5em; // ensure consistent height even if no certificate
+`
+
+const EllipsisTitle = styled(Dotdotdot)`
+  font-weight: bold !default;
```

Comment:
The `!default` bit here is invalid css; that's scss syntax.

Reply:
Removed.